src/pineko/cli/check_scalevar.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+import click
+import pineappl
+import rich
+
+from .. import check, theory_card
+from ._base import command
+
+
+@command.command("check_scalevar")
+@click.argument("pineappl_path", metavar="PINEAPPL", type=click.Path(exists=True))
+@click.argument("theory_ID", metavar="ID", type=click.Path(exists=True))
+def subcommand(pineappl_path, theory_ID):
```

Comment:
- let's keep this consistent with the rest of the "file-based" CLI: there is no theory - it just checks the availablibily
- if it fails print a nice message instead of an exception (as in the other check function)